tools/format.py

```diff
@@ -45,7 +45,7 @@ def main():
 
 def clang_format():
     print "clang_format"
-    exe = os.path.join(third_party_path, "depot_tools", "clang-format")
+    exe = get_buildtools_tool_path("clang-format")
```

Comment:
Refs https://github.com/denoland/deno/pull/2943#discussion_r324442881 (using option 3 here):

Digging through the `depot_tools` sources I've found out why this is necessary here. The wrapper script [expects `buildtools` to be at the top level](https://github.com/denoland/deno_third_party/blob/4b36ce22a2ef7310f815acb6a2de0d26766a24b6/depot_tools/gclient_paths.py#L101), but since we moved the symlink this is no longer the case.

So we have to either:
* set `CHROMIUM_BUILDTOOLS_PATH` explicitly or
* maintain 2 `buildtools` symlinks (one at the old top level position and one inside `core/libdeno` or
* calling `third_party/v8/buildtools/{win,mac,linux64}/clang-format` directly